src/lambda_codebase/initial_commit/bootstrap_repository/adf-bootstrap/deployment/lambda_codebase/pipeline_management/generate_pipeline_inputs.py

```diff
@@ -140,6 +140,14 @@ def generate_pipeline_inputs(
             list(target_structure.generate_waves()),
         )
 
+    # Print out if final target list is empty...
+    number_targets_total = 0
+    for target in pipeline_object.template_dictionary["targets"]:
+        number_targets_total = number_targets_total + len(target)
+    LOGGER.info("We found %s targets total", number_targets_total)
+    if number_targets_total == 0:
+        LOGGER.info("We'll create an empty pipeline with no targets!")
```

Comment:
Can we change the language here to not log "We"

Reply:
```suggestion
    LOGGER.info("Total number of accounts found: %d", number_targets_total)
    if number_targets_total == 0:
        LOGGER.info("Creating empty pipeline as there were no targets found")
```